lmfdb/verify/mf_newspaces.py

```diff
@@ -318,10 +329,10 @@ def check_Skchi_dim_formula(self, rec, verbose=False):
         if rec['level'] < 3:
             dirchar = rec['level']
         else:
-            dirchar = DirichletGroup_conrey(rec['level'])[rec['conrey_indexes'][0]].sage_character()
+            dirchar = get_dirchar(rec['level'], rec['conrey_indexes'][0], rec['char_order'])
```

Comment:
The changes in this file - here and below  - are untested, since I didn't know how to test them. I'd be grateful if someone could show me how to run these verification scripts.

Reply:
Thanks @roed314 for showing me how to run these (and fixing them along the way)! The verification is not passing right now, which I'm getting to the bottom of

Reply:
Latest commit c321e21 resolves the verification test failures. For details of the issue see my comment below that commit.